src/views/FoodMain.js

```diff
@@ -2,9 +2,16 @@ import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import CardColumns from 'react-bootstrap/CardColumns';
 import FavoriteFoods from './FavoriteFoods';
-import { Form } from 'react-bootstrap/Form';
+import {Form}  from 'react-bootstrap';
 
+{/** So the above renders but this doesnt
+  import {Form}  from 'react-bootstrap/Form';
 
```

Comment:
@dahlbyk  any idea why this is?

Reply:
Not sure why it is, but I typically haven't been putting { } around my bootstrap components I'm bringing in, although I haven't tried using Form yet.

Reply:
To write comments in JSX, you need to use JavaScript's forward-slash and asterisk syntax, enclosed inside a curly brace {/* comment here */}

I was in the wrong spot

Reply:
J to the S to the X

Reply:
Sorry i should've been specific- i meant in response to your comment in the code about the import {Form}. I think you need { } if you're don't specify the react-bootstrap/Form path, but if you specify the /Form endpoint then you don't need the { }.